models/daily_spend.sql

```diff
@@ -143,14 +148,42 @@ cloud_services_spend_daily as (
     group by 1, 2, 3, 4
 ),
 
+credits_billed_daily as (
+    select
+        date,
+        sum(credits_used_cloud_services) as daily_credits_used_cloud_services,
+        sum(credits_used_cloud_services + credits_adjustment_cloud_services) as daily_billable_cloud_services
+    from {{ ref('stg_metering_daily_history') }}
+    where
+        service_type = 'WAREHOUSE_METERING'
+    group by 1
+),
```

Comment:
@NiallRees updated this CTE to use the actual billed instead of calculating it with 10%